src/Nri/Ui/Carousel/V1.elm

```diff
@@ -0,0 +1,94 @@
+module Nri.Ui.Carousel.V1 exposing (..)
+
+import Css exposing (..)
+import Html.Styled as Html exposing (Html)
+import Nri.Ui
+import Nri.Ui.Tooltip.V3 as Tooltip
+import TabsInternal.V2 as TabsInternal
+
+
+type Tab id msg
+    = Tab (TabsInternal.Tab id msg)
+
+
+type Attribute id msg
+    = Attribute (TabsInternal.Tab id msg -> TabsInternal.Tab id msg)
+
+
+type TabPosition
+    = Before
+    | After
+
+
+{-| Tooltip defaults: `[Tooltip.smallPadding, Tooltip.onBottom, Tooltip.fitToContent]`
+-}
+withTooltip : List (Tooltip.Attribute msg) -> Attribute id msg
+withTooltip attributes =
```

Comment:
Are we expecting to display tooltips on the carousel controls? If not, I'd love to delete this 💀

Reply:
Deleted! 🎉